config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Kaminari.configure do |config|
+  config.max_per_page = 100
```

Comment:
I found `max_pages` for some reason, does not behave as we expect. I set `max_pages` to 100 and created 150 records in database. `Projects.page(121).per(1)` still returns results. I was expecting it to return empty results.

Hence I went with the `before_action` approach which also has added advantage of not running any DB queries when page limit exceeds and returning error.